src/meshcore/commands/messaging.py

```diff
@@ -70,13 +71,49 @@ async def send_msg(
             timestamp = int(time.time())
 
         data = (
-            b"\x02\x00\x00"
+            b"\x02\x00"
+            + attempt.to_bytes(1, "little")
             + timestamp.to_bytes(4, "little")
             + dst_bytes
             + msg.encode("utf-8")
         )
         return await self.send(data, [EventType.MSG_SENT, EventType.ERROR])
 
+    async def send_msg_reliable(
+        self, contact, msg: str, timestamp: Optional[int] = None,
```

Comment:
for contact we should accept the DestinationType and run the validate on it so users can pass a name, pubkey or contact dict

Reply:
that was my first intention, then I moved to contact for simplicity, because I need it later

definitely should be destType

Reply:
destType would be the most consistent from api pov, after dest type returns a consistent pubkey we can always look it up in the contacts, to get the full contact